### ISSUE TITLE
GTZAN and CREMA-D

### ISSUE THREAD

**Issue by yunzqq** [nttcslab/m2d#2]
Hi,
Could you please provide the split setting on these two datasets?
Thank you!

**Comment by yunzqq** [comment]
> Hi, thanks for your interest.
> 
> You can find them in the evaluation package [EVAR](https://github.com/nttcslab/eval-audio-repr). Please follow: https://github.com/nttcslab/eval-audio-repr/blob/main/Preparing-datasets.md
> 
> Data splits can be found in evar/metadata/*.csv
> 
> Hope it helps.

Many thanks!

**Comment by daisukelab** [comment]
Please let us know if you publish your paper in the future!

**Comment by yunzqq** [comment]
> Please let us know if you publish your paper in the future!

hhh, OK!

**Comment by yunzqq** [comment]
> Please let us know if you publish your paper in the future!

May I ask another question? For the long audio recording, how long clips are made for training? and At inference time, the audio recording is split into clips and averaged the logit is used for final classification results?

Best,
Qiquan

**Comment by daisukelab** [comment]
> May I ask another question? For the long audio recording, how long clips are made for training? and At inference time, the audio recording is split into clips and averaged the logit is used for final classification results?

Thank you for your question! A quick answer is you would be correct.
- While pre-training, we randomly crop a fixed duration from training samples.
- At inference time, we do the following:

![image](https://github.com/nttcslab/m2d/assets/14831220/f74d7046-8b2d-424d-b621-0c9a3974e8e9)

And the runtime implementation is as follows:

https://github.com/nttcslab/m2d/blob/4cdffb04b3fb311fc64a87021ccc3cacf7a01ceb/m2d/runtime_audio.py#L173-L225

Please let me know if you have any more questions.

**Comment by yunzqq** [comment]
> 

Many thanks for your help！Are you in Greece for ICASSP 2023？If yes May I communicate with you ？ hahaha

**Comment by daisukelab** [comment]
Yes, I'm presenting this paper:
> AASP-P4: Anomaly Detection and Representation Learning for Audio Classification
> Room: Poster Area 2 - Garden  Type: Poster   03:35 PM to 5:05 PM
> 1773 (AASP-P4.3): MASKED MODELING DUO: LEARNING REPRESENTATIONS BY ENCOURAGING BOTH NETWORKS TO
MODEL THE INPUT

See you there! :)
